cetlib/get-accessible-paths.cc

```diff
@@ -7,7 +7,7 @@
 #include <vector>
 
 using namespace std::string_literals;
-namespace bfs = boost::filesystem;
+namespace bfs = std::filesystem;
```

Comment:
Retaining `bfs` as a name seems incongruous: could we replace it with `fs`, maybe?